quantus/metrics/faithfulness_metrics.py

```diff
@@ -188,6 +187,7 @@ def __call__(
                 model=model.get_model(), inputs=x_batch, targets=y_batch, **self.kwargs
             )
         a_batch = utils.expand_attribution_channel(a_batch, x_batch_s)
```

Comment:
same here - OR alternatively, we add a COMMENT on what these two lines 189 and 190 is doing?

Reply:
I think comments make sense

Reply:
ok super